src/containers/developer-page/index.tsx

```diff
@@ -27,11 +27,13 @@ export class DeveloperPage extends React.Component<Props> {
             <li>
               <Link to="/developer/">Developer</Link>
             </li>
+            <li>
+              <Link to="/widget-list/">Widget List</Link>
+            </li>
           </ul>
         </nav>
         <h1>Developer</h1>
         <button onClick={() => this.props.actions.createDashboard('Stanley Huang', 69)}>Hello</button>
-        <button onClick={() => this.props.actions.saveDashboard(69, 'Riley Gowanlock', {})}>Save</button>
```

Comment:
Just want to make sure we did want to delete this line.

Reply:
Lol no. Oops